# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintJob.h
# Commit: 4e05361c6b08
# Full Hash: 4e05361c6b08aad43b939c33f5aa608a3d9ccbc8
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p5 - Move nsPrintData::mPreparingForPrint to nsPrintJob. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146977
# ==============================================================================

diff -r cc4de7b25404 -r 4e05361c6b08 layout/printing/nsPrintJob.h
--- a/layout/printing/nsPrintJob.h	Sun May 22 09:37:56 2022 +0000
+++ b/layout/printing/nsPrintJob.h	Sun May 22 09:37:57 2022 +0000
@@ -268,6 +268,11 @@
 
   int32_t mNumPrintablePages = 0;
 
+  // If true, indicates that we have started Printing but have not gone to the
+  // timer to start printing the pages. It gets turned off right before we go
+  // to the timer.
+  bool mPreparingForPrint = false;
+
   bool mCreatedForPrintPreview = false;
   bool mIsCreatingPrintPreview = false;
   bool mIsDoingPrinting = false;
